### PR TITLE
vmui: remove send step param for instant queries

### PR DESCRIPTION
**Pull request by Loori-R**
#3896

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Review by hagen1778 — APPROVED** [review]
Thank you!
Let's wait review from @valyala

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): stop showing `Please enter a valid Query and execute it` error message on the first load of vmui.
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): properly process `Run in VMUI` button click in [VictoriaMetrics datasource plugin for Grafana](https://github.com/VictoriaMetrics/grafana-datasource).
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix the display of the selected value for dropdowns on `Explore` page.
+* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): remove send step param for instant queries. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3896).
```
Comment (valyala) [inline docs/CHANGELOG.md:27]:
```suggestion
* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): do not send `step` param for instant queries. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3896).
```

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@Loori-R , thanks for the pull request!

**Pull request merged**